tests/e2e/export-dfx-identity.test.js

```diff
@@ -0,0 +1,186 @@
+
+const safeCheckboxClick = async (page) => {
+  const safeCheckbox = await page.getByTestId('safe-checkbox', true);
+  await safeCheckbox.click();
+};
+
+const continueButtonClick = async (page) => {
+  const continueButton = await page.getByTestId('continue-button', true);
+  await continueButton.click();
+};
+
+const backButtonClick = async (page) => {
+  const backButton = await page.getByTestId('back-button', true);
+  await backButton.click();
+};
+
+const closeButtonClick = async (page) => {
+  const closeButton = await page.getByTestId('close-button', true);
+  await closeButton.click();
+};
+
+const isContinueButtonDisabled = async (page) => {
+  const isContinueButtonDisabledFunc = await page.$('[data-testid="continue-button"][disabled]') !== null;
+  expect(isContinueButtonDisabledFunc).toBe(true);
+};
+
+const checkIsSubtitleTextMatch = async (page) => {
+  const subtitle = await page.getByTestId('subtitle', true);
+  const subtitleText = await page.evaluate((el) => el.textContent, subtitle);
+  expect(subtitleText).toBe('In order to use your account in DFX, run the following command, using the .pem file download as route.');
+};
+
+const copyCommandIconButtonClick = async (page) => {
+  const copyCommandIconButton = await page.getByTestId('copy-button', true);
+  await copyCommandIconButton.click();
+};
+
+const accountSelectClick = async (page) => {
+  const accountSelect = await page.getByTestId('account-select', true);
+  await accountSelect.click();
+};
+
+const accountItemClick = async (page) => {
+  const accountItem = await page.getByTestId('select-item-Subaccount', true);
+  await accountItem.click();
+  await page.waitForTestIdSelector('select-item-Subaccount', { hidden: true });
+};
+
+const checkIsAccountNameMatch = async (page) => {
+  const accountName = await page.getByTestId('account-name', true);
+  const accountNameText = await page.evaluate((el) => el.textContent, accountName);
+  expect(accountNameText).toBe('Subaccount');
+};
+
+describe('Settings: Export DFX Identity', () => {
+  let browser;
+  let page;
+  const wrongPassword = '1234544444';
+
+  beforeAll(async () => {
+    browser = await setupChrome();
+
+    // Importing the account
+    page = await utils.createNewPage(browser);
+    await optionsPageUtils.importAccount(page, secrets.seedphrase, secrets.password);
+    await optionsPageUtils.unlock(page, secrets.password);
+
+    await page.close();
+  });
+
+  beforeEach(async () => {
+    page = await utils.createNewPage(browser);
+    await page.goto(chromeData.popupUrl);
+    await popupPageUtils.profileButtonClick(page);
+    await settingsButtonClick(page);
+    await exportDFXIdentityButtonClick(page);
+  });
+
+  afterEach(async () => {
+    await page.close();
+  });
+
+  afterAll(async () => {
+    await browser.close();
+  });
+
+  test('checking the continue button is disabled when password input is empty', async () => {
+    await safeCheckboxClick(page);
+    await continueButtonClick(page);
```

Comment:
When clicking on the back button, would be good if we add validation to check the user is returning to the right screen, what do you think? would you like to give it a try?

Reply:
This is not necessary to do every time you click on the back button since will be redundant, but just once would be good